src/minimega/vm.go

```diff
@@ -568,6 +581,8 @@ func (vm *BaseVM) Info(field string) (string, error) {
 		return vm.Namespace, nil
 	case "state":
 		return vm.State.String(), nil
+	case "statetime":
```

Comment:
Maybe `updated` makes more sense that `statetime` and instead of `time.Since`, report the actual time that the VM was mutated?